recipe/meta.yaml

```diff
@@ -75,22 +72,30 @@ outputs:
         - numpy
         - pip
         - setuptools
+        - astunparse
+        - expecttest
+        - hypothesis
+        - numpy
+        - psutil
         - pyyaml
         - requests
-        - future
-        - six
         - mkl-devel {{ mkl }}   # [blas_impl == "mkl"]
         - libcblas * *_mkl      # [blas_impl == "mkl"]
+        - setuptools
+        - types-dataclasses
+        - sympy
+        - filelock
+        - networkx
+        - jinja2
+        - fsspec
+        - packaging
+        - optree
         - libcblas              # [blas_impl != "mkl"]
         - liblapack             # [blas_impl != "mkl"]
-        - libgomp   # [linux]
-        - llvm-openmp    # [osx]
```

Comment:
why did you remove these C dependencies?

Reply:
I tried to follow this as close as possible https://github.com/pytorch/pytorch#from-source. I figured I could reintroduce anything extra only if needed.

Reply:
these are pretty specific to conda forge omp dependencies. please add them back